src/mocks/handlers.ts

```diff
@@ -170,6 +180,38 @@ export const handlers = [
     }
   }),
 
+  rest.patch('/v1/manufacturers/:id', async (req, res, ctx) => {
+    const body = (await req.json()) as EditManufacturer;
+
+    if (body.name === 'test_dup') {
+      return res(
+        ctx.status(409),
+        ctx.json({
+          detail:
+            'A manufacturer with the same name has been found. Please enter a different name',
+        })
+      );
+    }
+    if (body.name === 'Error 500') {
+      return res(ctx.status(500), ctx.json(''));
```

Comment:
This line is never run in unit tests, I think we need a test for the catchAllError requests the user to refresh like the other dialogs.

Similarly https://github.com/ral-facilities/inventory-management-system/pull/112/files#diff-bb09d52c957fd04a8ed1a9f64c0cd6da4bb6c207f92a0fa1bb81a0e957aeaf49L129 is also not run, but I think it can just be removed?

Reply:
Yes it can be removed. And I'll add a test to check for catchAllError requests